### PR TITLE
DBUS_SYSTEM_BUS_ADDRESS can be set in docker-compose.yml

### PR DESCRIPTION
**Pull request by zvin**
Change-type: patch

### CODE REVIEW

File: pages/learn/develop/runtime.md [hunk 1]
```diff
@@ -70,6 +70,18 @@ DBUS_SYSTEM_BUS_ADDRESS=unix:path=/host_run/dbus/system_bus_socket
 
 Below you can find a couple of examples. All of them requires either prepending the command with the above `DBUS_SYSTEM_BUS_ADDRESS=...` or setting the variable for all commands by running `export DBUS_SYSTEM_BUS_ADDRESS=...` with the correct environment variable value from above.
 
+In multicontainer applications, you can also set `DBUS_SYSTEM_BUS_ADDRESS=unix:path=/host/run/dbus/system_bus_socket` in the `environment` section of your service in the `docker-compose.yml` file:
+
+```
```
Comment (garethtdavies) [inline pages/learn/develop/runtime.md]:
```suggestion
```yaml
```

While we don't make use of these I've been adding these as a matter of styling improvements.

File: pages/learn/develop/runtime.md [hunk 2]
```diff
@@ -70,6 +70,18 @@ DBUS_SYSTEM_BUS_ADDRESS=unix:path=/host_run/dbus/system_bus_socket
...
+version: '2'
+services:
+  someservice:
+    environment:
+        - 'DBUS_SYSTEM_BUS_ADDRESS=unix:path=/host/run/dbus/system_bus_socket'
```
Comment (garethtdavies) [inline pages/learn/develop/runtime.md]:
```suggestion
      - 'DBUS_SYSTEM_BUS_ADDRESS=unix:path=/host/run/dbus/system_bus_socket'
```

Some extra whitespace there?

File: pages/learn/develop/runtime.md [hunk 3]
```diff
@@ -70,6 +70,18 @@ DBUS_SYSTEM_BUS_ADDRESS=unix:path=/host_run/dbus/system_bus_socket
...
+    labels:
+      io.balena.features.dbus: 1
```
Comment (garethtdavies) [inline pages/learn/develop/runtime.md]:
```suggestion
      io.balena.features.dbus: '1'
```

Just matching example from https://www.balena.io/docs/learn/develop/multicontainer/#labels

File: pages/learn/develop/runtime.md (hunk 1 above)
Comment (xginn8) [reply pages/learn/develop/runtime.md]: time for a styleguide laying this and other conventions out?

Comment (garethtdavies) [reply pages/learn/develop/runtime.md]: Agreed. We should probably just adopt something such as [Google's](https://developers.google.com/style), and obviously we can supplement our own product names and other balena specific items.

**Pull request merged**